ccm_web/client/src/App.tsx

```diff
@@ -1,18 +1,61 @@
 import React from 'react'
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
+import { BrowserRouter as Router, Link as RouterLink, Route, Switch } from 'react-router-dom'
+import { Breadcrumbs, Link, makeStyles, Typography } from '@material-ui/core'
 
 import ConsumerTest from './components/ConsumerTest'
-import Home, { mergeSectionProps } from './pages/Home'
-import MergeSections from './pages/MergeSections'
+import Home from './pages/Home'
+import { mergeSectionCardProps, gradebookToolsCardProps, createSectionsCardProps, addUMUsersCardProps, addNonUMUsersCardProps } from './components/FeatureCard'
 import './App.css'
 
+const useStyles = makeStyles((theme) => ({
+  breadcrumbs: {
+    paddingLeft: 25,
+    paddingTop: 25
+  }
+}))
+
+interface TitleTypographyProps {
+  to?: string
+}
+
 function App (): JSX.Element {
+  const classes = useStyles()
+  const featureCardProps = [mergeSectionCardProps, gradebookToolsCardProps, createSectionsCardProps, addUMUsersCardProps, addNonUMUsersCardProps]
   return (
     <div className='App'>
       <Router>
+        <div className={classes.breadcrumbs}>
+          <Route>
+              {({ location }) => {
+                const pathnames = location.pathname.split('/').filter(x => x)
+                return (
+                  <Breadcrumbs aria-label="breadcrumb">
+                    <Link component={RouterLink} to='/'>
+                      <Typography color='textPrimary'>
+                        Canvas Course Manager
+                      </Typography>
+                    </Link>
```

Comment:
BreadCrumbs look nice in terms of spacing and size. Accessibility/UX folks may encourage us to make the linked crumbs look noticeable in some way besides the hover underline. Do the wireframes give a specific separator to use? Looks like you can pass that to `Breadcrumbs`: https://material-ui.com/components/breadcrumbs/#custom-separator
<img width="418" alt="Screen Shot 2021-03-29 at 3 58 15 PM" src="https://user-images.githubusercontent.com/35741256/112897977-44eca200-90ae-11eb-8e87-e9aed0b2421c.png">

Reply:
At least one of the wireframes has a ">" separator but I overlooked that detail. I can certainly change that.